mzroll/tabledockwidget.cpp

```diff
@@ -2389,3 +2282,155 @@ MatrixXf TableDockWidget::getGroupMatrix(vector<mzSample*>& samples, PeakGroup::
     return X;
 }
 
+QWidget* TableToolBarWidgetAction::createWidget(QWidget *parent) {
```

Comment:
@GeorgeSabu This is a virtual function of class QWidgetAction. Get's called when an instance of this class or class inheriting this class is made.